src/test/java/giis/eshopcontainers/e2e/functional/CatalogTests.java

```diff
@@ -0,0 +1,59 @@
+package giis.eshopcontainers.e2e.functional;
+
+import giis.eshopcontainers.e2e.functional.common.BaseLoggedClass;
+import giis.eshopcontainers.e2e.functional.common.ElementNotFoundException;
+import giis.eshopcontainers.e2e.functional.utils.Click;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+import static giis.eshopcontainers.e2e.functional.utils.Shopping.*;
+
+class CatalogTests extends BaseLoggedClass {
+    @Test
+    @DisplayName("AddProductsToBasket")
+    void addProductsToBasket() throws ElementNotFoundException {
```

Comment:
This test makes different steps (login, add to basket, logout). Readability would be improved if logical blocks are separated by a blank line